libs/api_common/api_common.cpp

```diff
@@ -18,19 +18,22 @@
 #include "ebpf_verifier.hpp"
 #pragma warning(pop)
 
+thread_local static const ebpf_program_type_t* _global_program_type = nullptr;
+thread_local static const ebpf_attach_type_t* _global_attach_type = nullptr;
```

Comment:
Ok for now but globals (and thread locals) are usually considered bad programming practice.
Is it easy to remove these in the future?

Reply:
This thread_local is used by get_program_type_windows() callback from the verifier code. We can get rid of this global if the callback functions in g_ebpf_platform_windows also include a context pointer, which can then be used to store/fetch this information.

Reply:
Ok, something to keep in mind for later.